unison-src/transcripts/test-command.output.md

```diff
@@ -85,60 +96,24 @@ test2 = [Ok "test2"]
 
     New test results:
   
-  ◉ lib.test2   test2
+  ◉ lib.testInLib   testInLib
   
   ✅ 1 test(s) passing
   
-  Tip: Use view lib.test2 to view the source of a test.
-
-```
-`test` will descend into namespaces named `lib` if they aren't at the top-level, though.
+  Tip: Use view lib.testInLib to view the source of a test.
 
-```unison
-test3 : [Result]
-test3 = [Ok "test3"]
-test4 = [Ok "test4"]
 ```
+`test` WILL run tests within `lib` if ucm is cd'd inside.
 
 ```ucm
+.lib> test
 
-  I found and typechecked these definitions in scratch.u. If you
-  do an `add` or `update`, here's how your codebase would
-  change:
-  
-    ⍟ These new definitions are ok to `add`:
-    
-      test3 : [Result]
-      test4 : [Result]
-
-```
-```ucm
-.> test
-
-    
-    Cached test results (`help testcache` to learn more)
-    
-    ◉ test1   test1
-    
-    ✅ 1 test(s) passing
-    
-    ✅  
-
-  
-
-  
-
-  
-
-  
-
-    New test results:
+  Cached test results (`help testcache` to learn more)
   
-  ◉ hello.lib.test3   test3
```

Comment:
These were showing up before when I don't believe they should've been.
The reason being: the old name filtering code only filtered out names which begin with `lib`, but IMO, we want to filter out everything within a lib regardless where in the tree it is. The new branch filtering method I use in this PR makes this possible whereas with name filtering it's very expensive.

This PR removes these lines from the transcript output, but also rewrites parts of the transcripts to clarify the behaviour here.